src/ontology/mondo-ingest.Makefile

```diff
@@ -369,44 +367,43 @@ deploy-mondo-ingest:
 	ls -alt $(DEPLOY_ASSETS_MONDO_INGEST)
 	gh release create $(GHVERSION) --notes "TBD." --title "$(GHVERSION)" --draft $(DEPLOY_ASSETS_MONDO_INGEST)
 
-USE_MONDO_RELEASE=false
 
+# make function, not target! 
 # Builds tmp/mondo/ and rebuilds mondo.owl and mondo.sssom.tsv, and stores hash of latest commit of mondo repo main branch in tmp/mondo_repo_built
-tmp/mondo_repo_built:
-	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
-		rm -rf ./mondo/ &&\
-		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
-		cd mondo/src/ontology &&\
-		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
-		latest_hash=$$(git rev-parse origin/master) &&\
-		echo "$$latest_hash" > $@ &&\
-		cp $@ mappings/mondo.sssom.tsv mondo.owl ../../../; fi
-
-# Triggers a refresh of tmp/mondo/ and a rebuild of mondo.owl and mondo.sssom.tsv, only if mondo repo main branch has new commits
-.PHONY: refresh-mondo-clone
-refresh-mondo-clone:
-	@if [ ! -d tmp/mondo ]; then \
-		$(MAKE) tmp/mondo_repo_built -B; \
+
+define build_mondo
+	cd $(TMPDIR) && \
+	rm -rf ./mondo/ && \
+	git clone --depth 1 https://github.com/monarch-initiative/mondo && \
+	cd mondo/src/ontology && \
+	make mondo.owl mappings -B MIR=false IMP=false MIR=false \
+	latest_hash=$$(git rev-parse origin/master) && \
+	cd ../../../.. && \
+	echo "$$latest_hash" > $(1)
+endef
+
+tmp/mondo_repo_built: .FORCE
+	@if [ ! -f $@ ]; then \
+		$(call build_mondo, $@); \
 	else \
-		current_hash=$$(cat tmp/mondo_repo_built); \
+		current_hash=$$(cat $@); \
 		cd tmp/mondo; \
 		git fetch origin; \
 		latest_hash=$$(git rev-parse origin/master); \
 		if [ ! "$$current_hash" = "$$latest_hash" ]; then \
 			cd .. && mv mondo mondo-bak && mv mondo_repo_built mondo_repo_built-bak; \
-			cd .. && $(MAKE) tmp/mondo_repo_built -B; \
+			cd .. && $(call build_mondo, $@); \
 			rm -rf tmp/mondo-bak tmp/mondo_repo_built-bak; \
 		fi; \
 	fi
 
-.PHONY: up-to-date-mondo.sssom.tsv
-up-to-date-mondo.sssom.tsv: refresh-mondo-clone
+$(TMPDIR)/mondo.owl: tmp/mondo_repo_built
+	cp $(TMPDIR)/mondo/src/ontology/mondo.owl $@
 
-.PHONY: up-to-date-mondo.owl
-up-to-date-mondo.owl: refresh-mondo-clone
+ $(TMPDIR)/mondo.sssom.tsv: tmp/mondo_repo_built
+	cp $(TMPDIR)/mondo/src/ontology/mappings/mondo.sssom.tsv $@
 
-reports/mirror_signature-mondo.tsv:
-	$(MAKE) up-to-date-mondo.owl
+reports/mirror_signature-mondo.tsv: tmp/mondo.owl
```

Comment:
Should `tmp/mondo.owl` be `$(TMPDIR)/mondo.owl`?

Reply:
There are 29 occurrences of this in the `Makefile`.. I will do a refactor after this PR is merged.